rust/feature-flags/src/cohort/cohort_cache_manager.rs

```diff
@@ -93,25 +94,25 @@ impl CohortCacheManager {
         // If we get here, we have a cache miss
         common_metrics::inc(
             COHORT_CACHE_MISS_COUNTER,
-            &[("team_id".to_string(), team_id.to_string())],
+            &[("project_id".to_string(), project_id.to_string())],
             1,
         );
 
         // Attempt to fetch from DB
-        match Cohort::list_from_pg(self.reader.clone(), team_id).await {
+        match Cohort::list_from_pg(self.reader.clone(), project_id).await {
```

Comment:
this is the reason I changed the relationship – rather than looking up the team ID directly when getting cohort relationships, I want to look up cohorts that match the project_id <-> team_id relationship.  Figured it made more sense to just index cohorts by project_id everywhere.